src/util.js

```diff
@@ -32,10 +32,15 @@ export const flattenDeep = (list /* : any[] */) /* : any[] */ =>
     list.reduce((memo, x) => memo.concat(Array.isArray(x) ? flattenDeep(x) : x), []);
 
 const UPPERCASE_RE = /([A-Z])/g;
-const MS_RE = /^ms-/;
-
-const kebabify = (string /* : string */) /* : string */ => string.replace(UPPERCASE_RE, '-$1').toLowerCase();
-export const kebabifyStyleName = (string /* : string */) /* : string */ => kebabify(string).replace(MS_RE, '-ms-');
+const UPPERCASE_RE_TO_KEBAB = (match /* : string */)  /* : string */ => `-${match.toLowerCase()}`;
+
+export const kebabifyStyleName = (string /* : string */) /* : string */ => {
+  const result = string.replace(UPPERCASE_RE, UPPERCASE_RE_TO_KEBAB);
+  if (result.slice(0, 3) === 'ms-') {
```

Comment:
Is this the fastest way to check this? It seems like checking `result[0] === "ms" && result[1] === "s" && result[2] === "-"` might be faster than slicing the string, but is also really ugly. Maybe that's what [`.startsWith()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) does, but not everything supports it yet...